src/js/common/components/form-elements/ErrorableFileInput.jsx

```diff
@@ -54,7 +54,14 @@ class ErrorableFileInput extends React.Component {
             {requiredSpan}
           </label>
           {errorSpan}
-          <label role="button" tabIndex="0" htmlFor={this.inputId} className="usa-button usa-button-secondary">{this.props.buttonText}</label>
+          <label
+            role="button"
+            tabIndex="0"
+            htmlFor={this.inputId}
+            aria-describedby={this.props['aria-describedby']}
+            className={this.props.triggerClass || 'usa-button usa-button-secondary'}>
```

Comment:
Should we use classNames to manage the logic here if we have any classes we always plan to apply, or do we anticipate cases in which we would not some of the default classes applied?

Reply:
I think this ok when we're just overriding with an option prop. If it gets more complicated, we should definitely use classNames.